Makefile

```diff
@@ -2,13 +2,15 @@
 
 .PHONY: prereqs build test test_fmt validate-generated fmt lint ci devenv
 
+VERSION=`git describe --always --long --dirty`
```

Comment:
@weinong Yes it was on purpose. Cole and I discussed and felt like this should be sufficient. I don't feel strongly either way...